h2o-py/h2o/group_by.py

```diff
@@ -17,25 +17,47 @@ class GroupBy(object):
     """
     A class that represents the group by operation on an H2OFrame.
 
+    The returned groups are sorted by the natural group-by column sort.
+
+    :param H2OFrame fr: H2OFrame that you want the group by operation to be performed on.
+    :param by: by can be a str, a list of str, an int or a list of int which denote the column name(s) or
```

Comment:
either a name or an index of a single column, or a list for multiple columns -- denote the set of columns to group by.